sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache.cpp

```diff
@@ -10,6 +10,9 @@
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
+// RUN: %{build} -o %t_vnni.out -DVNNI -ffp-model=precise
```

Comment:
row_major test is currently disabled for SG32: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/SG32/joint_matrix_rowmajorA_rowmajorB.cpp

Did you check that this test is passing with SG32 and -DVNNI?

Reply:
Correct, I will remove the the row major run and add a TODO comment.